test/test_misc.jl

```diff
@@ -115,6 +136,41 @@ end
     end
 end
 
+@testset "performance hash eq" begin
+    rng = MersenneTwister(123)
+    pts1 = [randn(rng, 3) for _ in 1:10^3]
```

Comment:
Maybe comment why this needs to be random

Reply:
Does not need to be random, it is just a lazy way to get a thousand distinct points. You would prefer something else?

Reply:
I removed the Random test dependency

Reply:
"Generate some random ponits" would be fine  just so it's not mysterious